core/core_bind.h

```diff
@@ -119,8 +119,8 @@ class OS : public Object {
 
 public:
 	enum VideoDriver {
-		VIDEO_DRIVER_GLES2,
 		VIDEO_DRIVER_VULKAN,
+		VIDEO_DRIVER_OPENGL,
```

Comment:
`O` comes before `V` in the alphabet, there's no purpose to swapping the order here.

Reply:
I swapped the order to match the order found in the project settings' enum: https://github.com/Calinou/godot/blob/8fb7cafaeb31b9d27b199effbc8a1829a0fa4845/main/main.cpp#L1271